frontend/packages/kubevirt-plugin/src/components/form/k8s-resource-select-row.tsx

```diff
@@ -70,7 +70,7 @@ export const K8sResourceSelectRow: React.FC<K8sResourceSelectProps> = ({
         onChange={onChange}
         value={asFormSelectValue(nameValue)}
         id={id}
-        isDisabled={isDisabled || isLoading || loadError}
+        isDisabled={isDisabled || isLoading || !!loadError}
```

Comment:
FYI - TypeScript 3.7 introduces the [nullish coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing) language feature for handling values that might be `null` or `undefined`. This helps to separate ["falsy" values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) vs. `null` or `undefined` specifically.

```ts
let x = foo ?? bar();
// .. is equivalent to ..
let x = (foo !== null && foo !== undefined) ? foo : bar();
```

Reply:
I am not sure I understand how this applies here? This change is here just to satisfy the type which is boolean but we are passing a string